ThunderDome/ThunderDome/Program.cs

```diff
@@ -28,7 +28,8 @@ static void Main(string[] args)
             Console.WriteLine("Your second pick is " + roster[secondpickButton].name);
 
             //Put both fighters into the ThunderDome
-            monsterBlock.ThunderDome(roster[firstpickButton], roster[secondpickButton]);
+            arena a = new arena();
+            a.ThunderDome(roster[firstpickButton], roster[secondpickButton]);

```

Comment:
We might need to discuss to understand why I am suggesting this.  Your program should run the same but this is an example of IMHO better OOP.  First off, your monsterBlock class had thing that probably should not be part of the monsterBlock.  The monsterBlock and the arena they fight in seem to me to be two separate objects.  At this stage maybe you don't care about this but this is and example of good design patterns.   If you read about MVC you will  see that you want to separate the Model (ie your monster) from the Controller (ie the arena), the View is your Console.WriteLine stuff.  A red flag to me was when I say a monster method actually performed functions one two monster, attacker and target.   This is all a POV and I may have the wrong take on what you have in mind, so accept only if you agree but lets chat so you at least understand what I am getting at.